sqlmesh/utils/jinja.py

```diff
@@ -130,19 +132,19 @@ def find_call_names(node: nodes.Node, vars_in_scope: t.Set[str]) -> t.Iterator[t
             for arg in child_node.args:
                 vars_in_scope.add(arg.name)
         elif isinstance(child_node, nodes.Call):
-            name = call_name(child_node)
+            name = node_name(child_node)
             if name[0][0] != "'" and name[0] not in vars_in_scope:
-                yield name
+                yield (name, child_node)
```

Comment:
why did you change if it you don't use this on line 147?

Reply:
I use it in manifest.py